server/modules/auth/api/authApi.js

```diff
@@ -54,7 +57,9 @@ module.exports.init = app => {
     const { user } = req
     RecordService.terminateUserThread(user.id)
 
-    req.logout()
+    const token = req.headers.authorization.substring(7)
```

Comment:
can we add to Jwt a constant `Bearer ` so that we can use here (and anywhere Bearer is used) the constant? here it could be Jwt.bearerPrefix.length ?

Reply:
Ok